datarepo-clienttests/src/main/java/scripts/testscripts/SnapshotScaleDelete.java

```diff
@@ -0,0 +1,124 @@
+package scripts.testscripts;
+
+import bio.terra.datarepo.api.DataRepositoryServiceApi;
+import bio.terra.datarepo.api.RepositoryApi;
+import bio.terra.datarepo.client.ApiClient;
+import bio.terra.datarepo.model.DeleteResponseModel;
+import bio.terra.datarepo.model.JobModel;
+import bio.terra.datarepo.model.SnapshotModel;
+import bio.terra.datarepo.model.SnapshotSummaryModel;
+import com.google.cloud.storage.BlobId;
+import common.utils.StorageUtils;
+import java.util.ArrayList;
+import java.util.List;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import runner.config.TestUserSpecification;
+import scripts.testscripts.baseclasses.SimpleDataset;
+import scripts.utils.DataRepoUtils;
+
+public class SnapshotScaleDelete extends SimpleDataset {
+  private static final Logger logger = LoggerFactory.getLogger(SnapshotScaleDelete.class);
+
+  /** Public constructor so that this class can be instantiated via reflection. */
+  public SnapshotScaleDelete() {
+    super();
+  }
+
+  private SnapshotModel snapshotModel;
+
+  private static List<BlobId> scratchFiles = new ArrayList<>();
+  private String dirObjectId;
+
+  private int NUM_SNAPSHOTS = 1;
+
+  public void setParameters(List<String> parameters) {
+    if (parameters != null && parameters.size() > 0) {
+      NUM_SNAPSHOTS = Integer.parseInt(parameters.get(0));
+    }
+    logger.debug("Number of snapshots to create and delete (default is 1): {}", NUM_SNAPSHOTS);
+  }
+
+  public void setup(List<TestUserSpecification> testUsers) throws Exception {
+    // create the profile and dataset
+    super.setup(testUsers);
+
+    // get the ApiClient for the snapshot creator, same as the dataset creator
+    ApiClient datasetCreatorClient = DataRepoUtils.getClientForTestUser(datasetCreator, server);
+    RepositoryApi repositoryApi = new RepositoryApi(datasetCreatorClient);
+
+    // dont bother to load data into the new dataset
+    // create MANY snapshots based on a single dataset
+
+    /*    for (int i = 0; i < NUM_SNAPSHOTS; i++) {
```

Comment:
If the code isn't needed it would be better to remove it then comment it out, unless you plan to add it back in soon.